internal/pkg/vm/testing.go

```diff
@@ -213,6 +229,9 @@ func (ts testStorage) Commit(cid.Cid, cid.Cid) error {
 
 // Head returns an empty Cid to satisfy the Storage interface
 func (ts testStorage) Head() cid.Cid {
+	if ts.state == nil {
+		return cid.Undef
+	}
```

Comment:
@acruikshank any idea how some tests were passing before? 
`encoding.Encode({{ nil }}.state)` was blowing up during initactor tests.

Reply:
Which tests weren't passing with what changes? I don't know how to answer this question.

Reply:
`builtin/miner/miner_test.go:184` the `TestConstructor` added a ~week ago was not passing now if didn't add that `nil` check.